src/solvers/flattening/bv_pointers.h

```diff
@@ -16,6 +16,9 @@ Author: Daniel Kroening, kroening@kroening.com
 
 class bv_pointerst:public boolbvt
 {
+private:
+  typedef boolbvt supert;
+
```

Comment:
Unless there is really a burning need, I don't see that renaming things, especially if this requires wide, shallow changes, is worthwhile.

Reply:
`SUB` is misleading and doesn't respect coding standards.
For the first time reader of code mentioning `SUB`  it looks like a mysterious macro, while `supert` is clearly the super type of something.

Reply:
Except that C++ doesn't use a notion of "super types". Please refer to them as base classes, and hence, e.g., `baset`.

Reply:
@tautschnig done